app/src/test/java/nl/tudelft/bbw/APITest.java

```diff
@@ -47,13 +49,21 @@
     @Before
     public final void setUp() throws HashException, BlockAlreadyExistsException {
         EdDSAPrivateKey privateKey = ED25519.generatePrivateKey();
-        owner = new User("Jeff", "iban", ED25519.getPublicKey(privateKey));
+        owner = new Acquaintance("Jeff", "iban", ED25519.getPublicKey(privateKey), new ArrayList<List<Block>>());
```

Comment:
why do you use an arraylist here? I one of my previous comments you changed the arraylist into list for the multichain.